analytics/Makefile

```diff
@@ -32,10 +33,20 @@ lint:
 	@echo "============================="
 	@echo "=> All checks succeeded"
 
-test:
-	@echo "=> Running tests"
+unit-test:
+	@echo "=> Running unit tests"
 	@echo "============================="
-	$(POETRY) run pytest
+	$(POETRY) run pytest --cov=src
+
+e2e-test:
+	@echo "=> Running end-to-end tests"
+	@echo "============================="
+	$(POETRY) run pytest tests/integrations --cov-append=src
+
+test-audit: unit-test
+	@echo "=> Auditing test coverage"
```

Comment:
This is a tiny tiny nit but, when I first read this I was fairly confused about what Auditing test coverage meant. I didn't know if it was testing our test coverage suite.

Maybe something simpler, like "Run test coverage report", would be more clear?

Reply:
This is a good suggestion! I like the phrase you suggested. I'll update in the next push!

Reply:
Fixed in this commit: [refactor: Updates help text for make test-audit](https://github.com/HHS/simpler-grants-gov/pull/913/commits/a2d6ca8cb8afd9b67db2a2d54082d1aaa76d210b)